### ISSUE TITLE
`no-undefined-references`: Missing support for tableCell

### ISSUE THREAD

**Issue by AaronMoat** [remarkjs/remark-lint#325]
### Initial checklist

- [x] I read the [support docs](https://github.com/remarkjs/.github/blob/main/support.md)
- [x] I read the [contributing guide](https://github.com/remarkjs/.github/blob/main/contributing.md)
- [x] I agree to follow the [code of conduct](https://github.com/remarkjs/.github/blob/main/code-of-conduct.md)
- [x] I searched [issues](https://github.com/search?q=user%3Aremarkjs&type=issues) and [discussions](https://github.com/orgs/remarkjs/discussions) and couldn’t find anything (or linked relevant results below)

### Affected package

remark-lint-no-undefined-references

### Steps to reproduce

Tables (with remark-gfm) don't appear to be supported by this lint rule. Should this be in scope for the package?

### Actual behavior

No linting failures for this file:

```
| Header  |
| :-----: |
| [stuff] |
```

### Expected behavior

An error reporting the undefined reference `[stuff]` should be reported

**Comment by wooorm** [comment]
Released in [`remark-lint-no-undefined-references@5.0.2`](https://github.com/remarkjs/remark-lint/releases/tag/remark-lint-no-undefined-references%405.0.2), thank you!
